en/hardware/porting_guide.md

```diff
@@ -15,7 +15,12 @@ Board startup and configuration files are located under [/boards](https://github
 For example, for FMUv5:
 * (All) Board-specific files: [/boards/px4/fmu-v5](https://github.com/PX4/Firmware/tree/master/boards/px4/fmu-v5). 
 * Build configuration: [/boards/px4/fmu-v5/default.cmake](https://github.com/PX4/Firmware/blob/master/boards/px4/fmu-v5/default.cmake).
-  
+* Board-specific initialisation file: [/boards/px4/fmu-v5/init/rc.board](https://github.com/PX4/Firmware/blob/master/boards/px4/fmu-v5/init/rc.board)
```

Comment:
@dagar @dagar This adds info on board-specific config from https://github.com/PX4/Firmware/pull/10960#issuecomment-443738341

I just found a good place to put the information and paraphrased you. 

The rest of the changes are trivial/layout and can be ignored.